### PR TITLE
Add missing msm8974 devices to readme

### PR DESCRIPTION
**Pull request by z3ntu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,8 +102,10 @@ page on the EFIDroid wiki for an exact mapping of LK targets to SoCs.
 - Fairphone 2 - FP2
 - LG G3 - D855
 - LG Google Nexus 5 - hammerhead D820, D821
+- OnePlus One - bacon (use `lk2nd-appended-dtb.img`)
```
Comment (wonderfulShrineMaidenOfParadise) [inline README.md:105]: Do you mean `lk2nd-msm8974-appended-dtb.img `?

Comment (z3ntu) [reply README.md:105]: I guess, when built from source tree it's just called `lk2nd-appended-dtb.img` though, just differs for GH releases

**Comment by z3ntu** [comment]
@TravMurav @stephan-gh Review please :)

**Comment by TravMurav** [comment]
Seems like I missed it earlier, thanks!

**Pull request merged**